### PR TITLE
feat(frontend): create SendModal with initial step of choosing the token

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

The following is the generic `SendModal` that has as first step the choice of token. The rest will be the same as the modals for ETH and IC.

### CODE REVIEW

File: src/frontend/src/lib/components/send/SendModal.svelte [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	import { waitWalletReady } from '$lib/services/actions.services';
+	import { loadTokenAndRun } from '$icp/services/token.services';
+	import { addressNotLoaded } from '$lib/derived/address.derived';
+	import { closeModal } from '$lib/utils/modal.utils';
+	import { ProgressStepsSend } from '$lib/enums/progress-steps';
+	import type { Network, NetworkId } from '$lib/types/network';
+	import SendTokensList from '$lib/components/send/SendTokensList.svelte';
+	import SendWizard from '$lib/components/send/SendWizard.svelte';
+
+	export let destination = '';
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/send/SendModal.svelte:19]: Same as existing code or anything new here?

File: src/frontend/src/lib/components/send/SendModal.svelte [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+	bind:this={modal}
+	on:nnsClose={close}
+	disablePointerEvents={currentStep?.name === WizardStepsSend.SENDING}
+>
+	<svelte:fragment slot="title">{currentStep?.title ?? ''}</svelte:fragment>
+
+	{#if currentStep?.name === WizardStepsSend.TOKENS_LIST}
+		<SendTokensList on:icSendToken={nextStep} />
+
+		<button class="secondary full center text-center" on:click={modalStore.close}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/send/SendModal.svelte:79]: This button can be included in `SendTokensList`?

File: src/frontend/src/lib/config/send.config.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { WizardStepsSend } from '$lib/enums/wizard-steps';
+import type { WizardSteps } from '@dfinity/gix-components';
+
+export const sendWizardSteps = (i18n: I18n): WizardSteps => [
+	{
+		name: WizardStepsSend.TOKENS_LIST,
+		title: i18n.send.text.send
+	},
+	{
+		name: WizardStepsSend.SEND,
```
Comment (peterpeterparker) [inline src/frontend/src/lib/config/send.config.ts:10]: I guess it just duplicates temporarly the similar existing steps right?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/config/send.config.ts:10]: correct! until we merge into one (another PR) or we use one of them to create the others

Comment (peterpeterparker) [reply src/frontend/src/lib/config/send.config.ts:10]: Great. Can you add a TODO?

File: src/frontend/src/lib/components/send/SendModal.svelte (hunk 2 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/send/SendModal.svelte:79]: sure! done

File: src/frontend/src/lib/components/send/SendModal.svelte (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/components/send/SendModal.svelte:18]: Great, then I do not review it.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/config/send.config.ts (hunk 3 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/config/send.config.ts:11]: Done:

- PR #1944

**Pull request merged**